medusa/network_timezones.py

```diff
@@ -67,17 +67,26 @@ def update_network_dict():
     network_list = dict(cache_db_con.select('SELECT * FROM network_timezones;'))
 
     queries = []
-    for network, timezone in iteritems(d):
+    remote_network_list = iteritems(d)
```

Comment:
@p0psicles also i use in two places so there is no reason to use iteritems(d) in both